libfwupdplugin/fu-quirks.c

```diff
@@ -248,7 +248,7 @@ fu_quirks_add_quirks_for_path(FuQuirks *self, XbBuilder *builder, const gchar *p
 	if (dir == NULL)
 		return FALSE;
 	while ((tmp = g_dir_read_name(dir)) != NULL) {
-		if (!g_str_has_suffix(tmp, ".quirk")) {
+		if (!g_str_has_suffix(tmp, ".quirk") && !g_str_has_suffix(tmp, ".quirk.gz")) {
```

Comment:
Presumably you still want to allow uncompressed quirks too. So perhaps want both checks.

Reply:
ohh it's saying "if not .quirk and not .quirk.gz" then skip this file. Is my logic wonky?